styles.css

```diff
@@ -75,12 +86,12 @@ header {
 	top: 50%;
 	left:50%;
 	transform: translate(-50%, -50%);
-	border: 2px solid;
 	order: 1;
-	padding: 5px 100px;
 	font-size: 4rem;
+	padding: 5px 100px;
 	text-transform: uppercase;
 	white-space: nowrap;
+	border: 2px solid;
 }
 
 main {
```

Comment:
display:grid means that this is not scaling well, consider using flex and perhaps flex-grow or flex-grow